common/components/WalletDecrypt/LedgerNano.scss

```diff
@@ -0,0 +1,26 @@
+.LedgerDecrypt {
```

Comment:
Styles ported from `Trezor.scss`. We could probably refactor to one shared file if that would make more sense.

Reply:
Could be, Digital Bitbox also looks _pretty_ similar, though there are differences. Might just be best keeping them custom so that we don't have to test all 3 wallets every time we make a change.